### PR TITLE
useAccessToken -> forAccessToken

### PR DESCRIPTION
**Pull request by yuhanniu**

### CODE REVIEW

File: src/Info.plist [hunk 1]
```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0.100</string>
+	<string>1.0.101</string>
```
Comment (sibinlu) [inline src/Info.plist]: do not bump version manually.

File: src/api/TKMember.h [hunk 2]
```diff
@@ -16,7 +16,9 @@
 #import "Money.pbobjc.h"
 #import "TransferTokenBuilder.h"
 #import "TKBrowser.h"
+#import "TKRepresentable.h"
 #import "TokenCluster.h"
+#import "TKRepresentable.h"
```
Comment (sibinlu) [inline src/api/TKMember.h]: duplicated header?

File: src/api/TKMember.h [hunk 3]
```diff
@@ -60,18 +62,11 @@
 - (TKClient *)getClient;
 
 /**
- * Sets the On-Behalf-Of authentication value to be used
- * with this client. The value must correspond to an existing
- * Access Token ID issued for the client member.
+ * Creates a member that acts as another member using an access token.
```
Comment (sibinlu) [inline src/api/TKMember.h]: Creates a `representable`

File: src/api/TKMemberSync.h [hunk 4]
```diff
@@ -58,18 +59,11 @@
 + (TKMemberSync *)member:(TKMember *)delegate;
 
 /**
- * Sets the On-Behalf-Of authentication value to be used
- * with this client.  The value must correspond to an existing
- * Access Token ID issued for the client member.
+ * Creates a member that acts as another member using an access token.
```
Comment (sibinlu) [inline src/api/TKMemberSync.h]: ditto

File: src/api/TKMemberSync.h [hunk 5]
```diff
@@ -104,7 +98,7 @@
 /**
  * Removes all public keys that do not have a corresponding private key stored on
  * the current device from tke member.
- *
+ *x
```
Comment (sibinlu) [inline src/api/TKMemberSync.h]: x?

File: src/api/TKMemberSync.m [hunk 6]
```diff
@@ -6,6 +6,7 @@
 #import "TKAccountSync.h"
 #import "TKMemberSync.h"
 #import "TKMember.h"
+#import "TKRepresentableSync.h"
```
Comment (sibinlu) [inline src/api/TKMemberSync.m]: You have added it to TKMemberSync.h, duplicated

File: src/api/TKRepresentable.h [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+ * The class provides async API
+ * </p>
+ */
+@protocol TKRepresentable
+
+/**
+ * Looks up funding bank accounts linked to Token.
+ *
+ * @param onSuccess callback invoked on success
+ * @param onError callback invoked on error
```
Comment (sibinlu) [inline src/api/TKRepresentable.h]: keep the param comments in same style

 * param accountId account id
 * param onSuccess callback invoked on success
 * param onError callback invoked on error

File: src/rpc/TKClient.m [hunk 8]
```diff
@@ -55,17 +55,33 @@ - (id)initWithGateway:(GatewayService *)gateway_
...
+    
+    if (self) {
+        gateway = gateway_;
+        crypto = crypto_;
+        memberId = memberId_;
+        rpc = rpc_;
+        errorHandler = errorHandler_;
+        unauthenticatedClient = unauthenticatedClient_;
+    }
+
```
Comment (sibinlu) [inline src/rpc/TKClient.m]: remove line

File: src/rpc/TKClient.m [hunk 9]
```diff
@@ -55,17 +55,33 @@ - (id)initWithGateway:(GatewayService *)gateway_
...
 }
 
+- (id)initWithRpc:(TKRpc *)rpc_
+          gateway:(GatewayService *)gateway_
+           Crypto:(TKCrypto *)crypto_
+         memberId:(NSString *)memberId_
+           client:(TKUnauthenticatedClient *) unauthenticatedClient_
+     errorHandler:(TKRpcErrorHandler *)errorHandler_ {
+    self = [super init];
+
```
Comment (sibinlu) [inline src/rpc/TKClient.m]: remove

File: src/api/TKRepresentable.h (hunk 7 above)
Comment (yuhanniu) [reply src/api/TKRepresentable.h]: done

**Pull request merged**